### PR TITLE
Catch exceptions in pointer handling

### PR DESCRIPTION
**Pull request by Hixie**
If we don't catch these exceptions, we get confused about what's going
on with the pointers, and the app basically stops working.

### CODE REVIEW

File: packages/flutter/lib/src/rendering/debug.dart [hunk 1]
```diff
@@ -11,7 +10,8 @@ import 'dart:ui' as ui;
 import 'package:flutter/painting.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/scheduler.dart';
-import 'package:vector_math/vector_math_64.dart';
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/debug.dart]: I don't understand why this wasn't complaining before. The code I removed doesn't refer to it as far as I can tell.

**Comment by Hixie** [comment]
Fixes https://github.com/flutter/flutter/issues/896.

**Comment by Hixie** [comment]
@abarth

**Comment by abarth** [comment]
LGTM

**Pull request merged**